src/app/store/slices/storage/storage.thunks/fetchSortedFolderContentThunk.ts

```diff
@@ -0,0 +1,85 @@
+import { ActionReducerMapBuilder, createAsyncThunk } from '@reduxjs/toolkit';
+import _ from 'lodash';
+
+import { storageActions } from '..';
+import { RootState } from '../../..';
+import { StorageState } from '../storage.model';
+import notificationsService, { ToastType } from '../../../../notifications/services/notifications.service';
+import { DriveItemData } from '../../../../drive/types';
+import { SdkFactory } from '../../../../core/factory/sdk';
+import { t } from 'i18next';
+import errorService from '../../../../core/services/error.service';
+
+const DEFAULT_LIMIT = 50;
+
+export const fetchSortedFolderContentThunk = createAsyncThunk<void, number, { state: RootState }>(
```

Comment:
Why haven't used the fetchPaginatedFolderContentThunk adding there the sort parameters?